### DISCUSSION TITLE
SkiaSharp.Views.Maui.Controls.SKCanvasView used for .Net MAUI

### DISCUSSION THREAD

**Discussion by TD65** [mono/SkiaSharp#2600] [Q&A]
Hi all, 
Please help resolving the error:
Microsoft.Maui.Platform.HandlerNotFoundException
  Message=Handler not found for view SkiaSharp.Views.Maui.Controls.SKCanvasView.
  Source=Microsoft.Maui
when using SkiaSharp libary in .Net MAUI project (.Net 7.0) as a process of converting Xamarin Forms code to .Net MAUI one? 
Thanks

**Comment by mattleibow** [comment]
Did you add the `.UseSkiaSharp()` in the maui program?

```cs
builder.UseSkiaSharp()
```

**Reply by TD65** [reply]
Hi,
Thanks for the reply. I added the builder.UseSkiaSharp() in the maui program, but in any class where I use SKCanvasView to draw content using code block:                      
  _canvasView = new SKCanvasView();                        
  _canvasView.PaintSurface += OnCanvasViewPaintSurface;
  Content = _canvasView;,
nothing happens, you know. OnCanvasViewPaintSurface is never called
 
The above code block runs well in Xamarin Forms, but in .Net MAUI it fails.
 
Regards

**Reply by taublast** [reply]
1.
`    <PackageReference Include="SkiaSharp.Views.Maui.Controls.Compatibility" Version="2.88.7" />`
2.
`builder.UseSkiaSharp(true);`

**Reply by rockthemeek** [reply]
have you found a solution..i've the same problem with the same version!

**Reply by mattleibow** [reply]
If this is happening, please open a new issue with a sample for me to run. I have been using this code just fine in all maui samples as well as some other libraries.
